Odata-docs/webapi-8/tutorials/basic-crud.md

```diff
@@ -311,6 +311,12 @@ In the controller, we add a contructor that accepts a `BasicCrudDbContext` param
 
 We will add controller actions to this controller progressively to support different CRUD operations.
 
+> [!IMPORTANT]
+> If you've made changes to the code, make sure that the controller's name matches the entity set's name registered in the entity data model (e. g. modelBuilder.EntitySet\<Customer>("**Customers**") matches **Customers**Controller).
```

Comment:
I think it’s helpful to know that a controller’s name should match the entity set’s name registered in the entity data model. However, this isn’t essential because the controller name can be overridden using the `[Route("api/Items")]` attribute.

For example, you can have the following controller:
```cs
[Route("api/Items")]
[ApiController]
public class HandleAllController : ODataController
{
    // Controller actions go here
}
```

Reply:
> the controller name can be overridden using the [Route("api/Items")] attribute

Agree, but in this case you need to specify a route for every endpoint with parameters, otherwise they won't work.